src/main/java/io/outfoxx/swiftpoet/CodeWriter.kt

```diff
@@ -447,26 +452,11 @@ internal class CodeWriter(
   /**
    * Returns the non-colliding importable types and module names for all referenced types.
    */
-  private fun generateImports(): Pair<Map<String, List<DeclaredTypeName>>, Set<String>> {
+  fun generateImports(): Pair<Map<String, List<DeclaredTypeName>>, Set<String>> {
     return importableTypes to referencedTypes.values.map { it.moduleName }.toSet()
   }
 
-  companion object {
```

Comment:
@kdubb I moved away from this style as it is a bit hard to carry additional information we need between the first pass and the second pass.

It was not being super helpful anyway and I did it initially to mirror KotlinPoet, but we are not KotlinPoet.